local-cli/init/init.js

```diff
@@ -113,7 +113,7 @@ function addJestToPackageJson(destinationRoot) {
   packageJSON.jest = {
     preset: 'react-native'
   };
-  fs.writeFileSync(packageJSONPath, JSON.stringify(packageJSON, null, '\t'));
+  fs.writeFileSync(packageJSONPath, JSON.stringify(packageJSON, null, '  '));
```

Comment:
You can pass `2` instead of 2 space characters in the 3rd param of JSON.stringify. It will make it a bit clearer.